includes/Settings.php

```diff
@@ -550,4 +552,4 @@ public function remove_notices() {
 	}
 }
 
-new File_Upload_Types_Settings();
+new \File_Upload_Types\Settings();
```

Comment:
IMO this should be removed as per my comment above.

Reply:
Also, as this call is done inside the `namespace File_Upload_Types;` namespace, there is no sense to providing an FQCN, so `new Settings()` would work the same and is shorter.